pybmix/utils/combinatorials.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
+
+class _stirling_memoizer(object):
```

Comment:
Typo (I guess): missing R in memoRizer

Reply:
this is actually correct, it is the name of a design pattern (memoizer)